src/conference_scheduler/resources.py

```diff
@@ -3,6 +3,62 @@
 
 
 class Slot(NamedTuple):
+    """A period of time at a venue in which an event can be scheduled
+
+    Parameters
+    ----------
+    venue : str
+        A human readable string
+    starts_at: datetime
+        The starting time for the time period
+    duration: int
+        The duration of the time period in minutes
+    capacity: int
+        This will be compared with :attr:`Event.demand` during computation of
+        the schedule to ensure that events are only scheduled in slots that can
+        accommodate them.
+    session: str
+        A human readable string which serves as a tag for similar time periods
+        e.g. 'morning', 'afternoon'
+
+    Example
+    -------
+    For a conference where:
+
+        * It will take place on 2016-09-17
+        * There are two rooms - 'Main Hall' and 'Small Room'
+        * The Main Hall can seat 500 people and the Small Room, 50
+        * It is intended to hold two 30 talks in the morning (from 09:30
+          to 10:00 and from 11:00 to 11:30) and two more in the afternoon
+          (from 14:00 to 14:30 and 15:00 to 15:30)
+
+    We would create the following eight objects::
+
+        Slot(
```

Comment:
We could doctest the doctstrings too? (Just include the `>>>` and `...` and make sure the doctest script also looks at the source files).